dockerfiles/docker-libv4l-build-cross-armhf/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM rust:1.36.0-buster
+
+ENV LIBCLANG_INCLUDE_PATH '/usr/include/clang/7/include'
+ENV PKG_CONFIG_PATH '/usr/lib/arm-linux-gnueabihf/pkgconfig'
+ENV PKG_CONFIG_ALLOW_CROSS 1
+
+ARG RUST_CHANNEL
+RUN test ${RUST_CHANNEL}
+
+WORKDIR /tmp/libv4l_sys
+
+RUN dpkg --add-architecture armhf \
+    && apt-get update -y \
+    && apt-get install -y libv4l-dev:armhf libclang-common-7-dev:armhf libclang-7-dev:armhf \
+    && apt-get install -y gcc-arm-linux-gnueabihf libclang-7-dev
+
+RUN rustup toolchain install ${RUST_CHANNEL} \
+    && rustup target add --toolchain ${RUST_CHANNEL} arm-unknown-linux-gnueabihf
+
+COPY . /tmp/libv4l_sys
```

Comment:
Is it necessary to copy everything of a directory?

Reply:
Not really.

When built with CI, this repository is cleanly checked out so copying everything is not a bad idea, I think.

But when built on developers' machine, files not committed to git are also copied. This might not be clean but I guess we can accept it (c.f. volume mount do the similar thing).

If you have any suggestion, please tell me about it 🙏 

- white list?
- black list w/ `.dockerignore` ?
    - which files should be ignored?

Reply:
In general, rust compiler generates `./target` directory and it grows to a large size (300MB~).
So at least, the directory should be ignored.